zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2943,6 +2981,32 @@ components:
           format: int64
           minimum: 1
 
+    SignalBroadcastRequest:
+      type: object
+      properties:
+        signalName :
+          description: The name of the signal to broadcast.
+          type: string
+        variables:
+          additionalProperties: true
+          description: The signal variables as a JSON object.
+          type: object
+        tenantId:
+          description: The id of the tenant that owns the signal.
```

```suggestion
          description: The ID of the tenant that owns the signal.
```

🔧 I'm pretty sure this is our preferred form of "ID", but at the very least it would be consistent within this PR.